iothub/device/tests/DeviceClientTests.cs

```diff
@@ -172,7 +172,7 @@ public async Task DeviceClient_OnMethodCalled_NullMethodRequest()
                 return Task.FromResult(new MethodResponse(Encoding.UTF8.GetBytes("{\"name\":\"ABC\"}"), 200));
             }, "custom data").ConfigureAwait(false);
 
-            await deviceClient.InternalClient.OnMethodCalled(null).ConfigureAwait(false);
+            await deviceClient.InternalClient.OnMethodCalledAsync(null).ConfigureAwait(false);
```

Comment:
I had these in my PR as well. Once this goes in, it'll decrease the size of my PR. I like that!